pyccl/nl_pt/bacco_lbias.py

```diff
@@ -0,0 +1,447 @@
+                `N_a` is the size of the object's `a_s` attribute.
+        """
+        self._check_init()
+
+        pk = self.lpt_table[0, :, :]
+
+        return pk*self.exp_cutoff
+
+    def get_biased_pk2d(self, tracer1, *, tracer2=None,
+                        extrap_order_lok=1, extrap_order_hik=2):
+        """Returns a :class:`~pyccl.pk2d.Pk2D` object containing
+        the PT power spectrum for two quantities defined by
+        two :class:`~pyccl.nl_pt.tracers.PTTracer` objects.
+
+        Args:
+            tracer1 (:class:`~pyccl.nl_pt.tracers.PTTracer`): the first
+                tracer being correlated.
+            tracer2 (:class:`~pyccl.nl_pt.tracers.PTTracer`): the second
+                tracer being correlated. If ``None``, the auto-correlation
+                of the first tracer will be returned.
+            extrap_order_lok (:obj:`int`): extrapolation order to be used on
+                k-values below the minimum of the splines. See
+                :class:`~pyccl.pk2d.Pk2D`.
+            extrap_order_hik (:obj:`int`): extrapolation order to be used on
+                k-values above the maximum of the splines. See
+                :class:`~pyccl.pk2d.Pk2D`.
+
+        Returns:
+            :class:`~pyccl.pk2d.Pk2D`: PT power spectrum.
+        """
+        if tracer2 is None:
+            tracer2 = tracer1
+
+        t1 = tracer1.type
+        t2 = tracer2.type
+
+        if t1 == 'IA' or t2 == 'IA':
+            raise ValueError("Intrinsic alignments not implemented in "
+                             "BaccoLbiasCalculator.")
+
+        if t1 == 'NC':
+            if t2 == 'NC':
+                pk = self._get_pgg(tracer1, tracer2)
+            else:  # Must be matter
+                pk = self._get_pgm(tracer1)
+        else:  # Must be matter
+            if t2 == 'NC':
+                pk = self._get_pgm(tracer2)
+            else:  # Must be matter
+                pk = self._get_pmm()
+
+        pk2d = Pk2D(a_arr=self.a_s,
+                    lk_arr=np.log(self.k_s),
+                    pk_arr=pk,
+                    is_logp=False,
+                    extrap_order_lok=extrap_order_lok,
+                    extrap_order_hik=extrap_order_hik)
+        return pk2d
+
+    def get_pk2d_template(self, kind, *, extrap_order_lok=1,
+                          extrap_order_hik=2):
+        """Returns a :class:`~pyccl.pk2d.Pk2D` object containing
+        the power spectrum template for two of the PT operators. The
+        combination returned is determined by ``kind``, which must be
+        a string of the form ``'q1:q2'``, where ``q1`` and ``q2`` denote
+        the two operators whose power spectrum is sought. Valid
+        operator names are: ``'m'`` (matter overdensity), ``'b1'``
+        (first-order overdensity), ``'b2'`` (:math:`\\delta^2`
+        term in galaxy bias expansion), ``'bs'`` (:math:`s^2` term
+        in galaxy bias expansion), ``'b3nl'`` (:math:`\\psi_{nl}`
```

Comment:
There is no b3nl in BACCO, right?

Reply:
True, I left it because I made sure that if you pass b3nl that term gets set to zero. But it's confusing, so now I left this behaviour unchanged (so if a b3nl gets passed it doesn't explode, just ignores it) but i changed the documentation so that b3nl doesn't appear as a legal parameter

Reply:
No, like in the lpt module the corresponding template is zero.

Reply:
ok, great